geonode_mapstore_client/static/mapstore/gn-translations/data.it-IT.json

```diff
@@ -360,7 +360,8 @@
             "maxLon": "Max Lon",
             "centerLat": "Lat",
             "centerLon": "Lon",
-            "mapExtentHelpText": "<b>Puoi modificare l'estensione e la posizione di questa risorsa</b>. Shift-clic e trascina per disegnare una nuova estensione, fare clic sul marcatore, quindi fare clic e trascinare per impostarne la posizione"
+            "mapExtentHelpText": "<b>Puoi modificare l'estensione e la posizione di questa risorsa</b>. Shift-clic e trascina per disegnare una nuova estensione, fare clic sul marcatore, quindi fare clic e trascinare per impostarne la posizione",
+            "unadvertised": "La risorsa non è pubblicizzata. È nascosto dal catalogo e dai risultati della ricerca"
```

Comment:
@dsuren1 small typo and let's use standard chars instead of UTF8:
`La risorsa non è pubblicizzata. E' nascosta dal catalogo e dai risultati della ricerca`